db/db.go

```
@@ -0,0 +1,52 @@
+package db
+
+//db global storage in memory
+var db = make(map[int64]Beer)
+
+//id auto incremental value
+var index int64 = 0
+
+//Create a new record in the database
+func Create(beer Beer) Beer {
+	index++
+	beer.ID = index
+	db[index] = beer
+	return beer
+}
+
+//Update a record by id in the database
+func Update(beer Beer, id int64) bool {
+	_, ok := db[id]
+
+	if !ok {
+		return ok
+	}
+
+	db[id] = beer
+
+	return ok
+}
+
+//Get a record by id in the database
+func Get(id int64) Beer {
```

Here we should return an error as well, for the user might not know if the Get operation failed.

```
func Get(id int64) (Beer, error){...
```

Returning an empty Beer when requesting one is not very happy... how would you feel if the bartender gave you an empty Beer when you order one? :rofl:  :nerd_face: 


_Note:_ Consider also the following, could be a different way to implement it:
```
func Get(id int64, beer *Beer) error{...
```